doc/liblouis.texi

```diff
@@ -2465,26 +2465,56 @@ tests:
   - [Good bye,  ⠠⠛⠕⠕⠙ ⠃⠽⠑]
 @end example
 
-The three basic components of a test file are as follows:
+The four basic components of a test file are as follows:
```

Comment:
I don't quite understand why there are four components now. AFAICT you did not add a component to the test file. If I understand correctly the `table:` component can now also handle a table query, but then there would still be three components. Ah I guess it's the `display`. OK, I get it now

Reply:
Yeah.